### DISCUSSION TITLE
Deployment failing

### DISCUSSION THREAD

**Discussion by historianem** [CollectionBuilder/collectionbuilder.github.io#90] [General]
Hi there, when I go to deploy my site (https://github.com/historianem/us-womenshistory-1865) on github I keep receiving the error message "github-pages can't satisfy your Gemfile's dependencies." The page deployment worked before today, and I don't believe I've made any significant changes besides adding files and updating my metadata csv, so I'm unsure what the issue is despite looking for it. Thanks in advance for your help!

**Comment by dcnb** [comment]
Hi @historianem!

I think you just need to delete the parentheses around the link in your include command on [line 11 in your pages/about.md file](https://github.com/historianem/us-womenshistory-1865/blob/549f4645ebd3547862d216863b87a08f2e9515d5/pages/about.md?plain=1#L11). 

So it should look like this: 

{% include feature/jumbotron.html objectid="https://upload.wikimedia.org/wikipedia/commons/0/09/Carciature_of_a_man-midwife_as_a_split_figure_Wellcome_L0018481.jpg" %}

Think that should fix it!

**Reply by historianem** [reply]
Ah! That did do it, thanks for your help!

**Comment by Bibliojo** [comment]
I am having the same issue . [build](https://github.com/Bibliojo/MiltonPLimages/actions/runs/5906572605/job/16023171482#step:4:37)
github-pages can't satisfy your Gemfile's dependencies. but I tsays that there is an error in Line 1

github-pages 228 | Error:  Invalid byte sequence in UTF-8 in line 1.
any advice greatly appreciated :)
jo

**Comment by dcnb** [comment]
Hi @Bibliojo , 

I'm thinking your issues have to do with the objects folder more generally -- CollectionBuilder doesn't display Tiff files on the web. (I don't think any browsers display them ...) So you'd need to convert those into jpgs or png files. You'll also want to make sure there are no spaces in the filenames of the images, and that you include the extension of any image in the filename field. 

So currently, your "item3" has a filename that reads "Milton PR3560 1832 F10"

That should be changed to something like "Milton_PR3560_1832_F10.jpg" and a jpg with that filename would need to be added to the "objects" folder in order for it to show up. 

Otherwise, it looks like your site is at least building, and the metadata is generating a map, timeline, and word clouds ... so that's something. 

Please feel free to follow up with more questions -- one of us will get to them next week. 

Cheers, 
@dcnb

**Comment by Bibliojo** [comment]
THANK YOU so much. How kind of you to respond. I did figure out that I mucked up the csv file ( really Jo how did you manage that?! ) and that was the first problem!
and yes I started to do some re-jigging but DUH I should not rush to get through something on a Friday afternoon!
so more to fix. . .
Your advice is fantastic thank you . pics are way too big, and files names need to be changed.
and I will get there, some days I think I'm just a really slow learner 😄
always trying to learn all the things!

have a super weekend!
thanks again
Jo



Joanne Paterson, BMus MA Musicology MScILS

Research and Scholarly Communication Librarian
Western Libraries
519.319.1488
OA Waivers and Discounts provided by Western Libraries<https://www.lib.uwo.ca/scholarship/oasupport.html>
Post-Prints in ***@***.*** meets OA Requirement by funder, at no cost!<https://ir.lib.uwo.ca/>
Ask me how!
Vacation Alert! September 7-28
________________________________

**Comment by Bibliojo** [comment]
Decided a do-over was in order :

https://bibliojo.github.io/MiltonParaLostImages/

Home, Paradise Lost Artwork from Weldon Milton collection<https://bibliojo.github.io/MiltonParaLostImages/>
Learning how to use CollectionBuilder-GH, a template for creating small digital collection exhibits on GitHub Pages designed for teaching digital library skills, to sget better and to show others this neat-o tool
bibliojo.github.io

Awesome, Now I feel comfortable showing others how to get started, and we can build something together 🙂
thanks again for you kind assistance!
Jo


Joanne Paterson, BMus MA Musicology MScILS

Research and Scholarly Communication Librarian
Western Libraries
519.319.1488
OA Waivers and Discounts provided by Western Libraries<https://www.lib.uwo.ca/scholarship/oasupport.html>
Post-Prints in ***@***.*** meets OA Requirement by funder, at no cost!<https://ir.lib.uwo.ca/>
Ask me how!
Vacation Alert! September 7-28
________________________________

**Comment by dcnb** [comment]
Very cool Jo! Nice work!

**Comment by Bibliojo** [comment]
[Image.png.png]

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
